dumpling/export/writer_util.go

```
@@ -160,42 +59,35 @@ func WriteInsert(
+	} else {
+		insertStatementPrefix = fmt.Sprintf("INSERT INTO %s VALUES\n",
+			wrapBackTicks(escapeString(meta.TableName())))
 	}
-	wp.currentFileSize += uint64(bf.Len())
+	var kinds []dumpformat.FieldKind
+	if selectedField != "" {
+		kinds = columnKinds(meta.ColumnTypes())
+	}
+	statementSize := int64(0)
+	if cfg.StatementSize != UnspecifiedSize {
+		statementSize = int64(cfg.StatementSize)
```

golangci-lint (2.12.2)

[high] G115: integer overflow conversion uint64 -> int64

(gosec)